### ISSUE TITLE
[Object] Everfrost Chip - Spawn/Loot/Quest

### ISSUE THREAD

**Issue by bongobg** [SunwellTracker/issues#748]
Decription: Everfrost Chip (https://db.darkwizard.pl/?object=193997) should be object scattered around Dun Niffelem and one of the ways to level reputation with Sons of Hodir. Looting it it should contain a Q starting item Everfrost Chip (https://db.darkwizard.pl/?item=44725) and once the Q is done it should give Everfrost Chip (https://db.darkwizard.pl/?item=44724) for turning in a repeatable Q. This second ID of Everfrost Chip should have a drop chance of 100%. The first issue is that there are not nearly enough spawns of the object and the second issue is that once the opening Q is done, the object does not give the second type of Everfrost Chips for the repeatable Q.

How it works: Exactly 4 spawns of the object and the loot table is not correct. This is know issue for Sunwell/Trinty based servers. Actually here I've seen only 1 of those 4 so not sure if there are 4 at all. Also even retail had some issues with this object's phasing.

How it should work: A lot of scattered spawns in larger area separated in 6 zones and 6 objects available at all time and 100% (66%) drop of the repeatable Q item after the opening Q is done.

Source (you should point out proofs of your report, please give us some source): 
Object map: http://www.wowhead.com/object=193997/everfrost-chip#map
Comments are explaining a lot.

**Comment by Dootz** [comment]
fixed

**Comment by Fixed7** [comment]
Tried it today, and it is not fixed. I can find the Chips, but I only get powder from it. Nadiir can confirm this, as I reported it to her in-game.

**Comment by Nadiirr** [comment]
Yeah. We tried 2 of them and it only dropped Everfrost Powder

**Comment by Dootz** [comment]
Have you completed quest http://www.wowhead.com/quest=13420/everfrost ?  
Which Chips doesn't drop:
- quest starting http://www.wowhead.com/item=44725/everfrost-chip
- or for repeatable quest http://www.wowhead.com/item=44724/everfrost-chip
?

**Comment by Fixed7** [comment]
@Dootz The one that starts the quest isn't dropping for me. I am friendly with Sons of hodir, but I have not looted my first chip to start the repeatable quest.

**Comment by Fixed7** [comment]
Another follow-up issue that was spotted by me today. I was in the area "The Storm Peaks" and I could see a chip, but when I try to collect it, I enter "Frostfield Lake" and it phases out. 

Proof: https://gyazo.com/22063fd6ceea0c8dd4b4c682d3aada81

**Comment by Dootz** [comment]
right now starting item have 20% drop chance. It'll be increased with next server update

**Comment by Dootz** [comment]
will be fixed with next server update (probably). Let me know if that helped. (around Tuesday/Wednesday)
